runtime/parallel/src/lib.rs

```diff
@@ -319,7 +319,7 @@ where
 }
 
 parameter_types! {
-    pub const PricePrecision: u8 = 3;
+    pub const PricePrecision: u8 = 6; // 18 - 12
```

Comment:
Can this information sent to runtime by client?

Reply:
It has been adjusted so that ocw-oracle's prices are in line with those of prices pallet.